api/config/routes.rb

```diff
@@ -133,6 +133,16 @@
       end
     end
 
+    resources :address_book, only: [], param: :address_id do
+      collection do
+        get :mine
+        post :update
+      end
+      member do
+        post :remove
+      end
+    end
```

Comment:
I don't get it. You don't use normal REST resource, but you pass params there. What exactly you want to do with that code?